### DISCUSSION TITLE
user friendly display name for a specific content type

### DISCUSSION THREAD

**Discussion by petrosmm** [directus/directus#15707] [Archive]
Hello,

Has the Directus team ever considered for the public good that tables (content types) named underneath might benefit from using a user friendly display name? Example: we had a content type (m2m relationship originating on the offices content type) called `rel_offices_to_employees_m2m`. The table seemed intuitive to name for the developers but to the user they see the following below. Is it worth considering that adding a field for each content type where you can override the display name, makes sense? This dogma is consistent with technical logic vs. business logic, for example, we would call it locations, they would call it offices.

![image](https://user-images.githubusercontent.com/8779484/191770495-884827bb-40e3-4552-a887-30a8f32ead57.png)

**Comment by azrikahar** [comment]
This is already possible! You can override it, or rather translate it, using the Field Name Translations and change it to any "display name" you'd like for any supported languages 🙂

![image](https://user-images.githubusercontent.com/42867097/191897490-56a400ae-9474-4609-b65e-481d7d365feb.png)

**Comment by petrosmm** [comment]
I feel so dumb. If only we would RTFM.
